addons/crm_project_issue/wizard/crm_lead2projectissue_wizard.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class CrmLeadToProjectIssueWizard(models.TransientModel):
+    """ wizard to convert a Lead into a Project Issue and move the Mail Thread """
+    _name = "crm.lead2projectissue.wizard"
+    _inherit = 'crm.partner.binding'
+
+    project_id = fields.Many2one('project.project', string='Project', domain=[('use_issues', '=', True)])
+
+    @api.multi
+    def action_lead_to_project_issue(self):
+        # get the models
+        Issue = self.env['project.issue']
+        for wizard in self:
+            self.model = self.env.context.get('active_model')
```

Comment:
make condition this way : 
if (self.env.context.get('active_model') == 'crm.lead') and self.env.context.get('active_id'):
            active_model = self.env['crm.lead'].browse(self.env.context.get('active_id'))